### PR TITLE
Tailored import - Main article

### PR DESCRIPTION
**Pull request by felix-couprie**
1st version of documentation

### CODE REVIEW

File: content/md/imports-exports/tailored-import.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+1. Click on `Create import profile`
+1. Type a unique export code and a label, then select the job `Tailored product import`<!-- or `Tailored product model import`-->. <!--You can select either CSV or XLSX for each job.-->
+![Create a new product import profile](../img/Tailored-Import_Create-import-profile.png)
+1. Save your new import profile
+
+# Edit a tailored product import profile
+
+Once you've created your tailored import profile, or if you want to edit an existing one:
+1. Go to `Imports`
+1. Click on the tailored import profile you want to edit
```
Comment (ghoussard) [inline content/md/imports-exports/tailored-import.md:41]:
```suggestion
1. Click on the Tailored Import profile you want to edit
```

File: content/md/imports-exports/tailored-import.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+<!--
+:::tips
+To learn more about Tailored Import and how to use this feature, please watch our training [Tailored Imports](https://akademy.akeneo.com/tailored-exports-foundations).
+:::
+-->
+
+# Create a tailored product import profile
+
+To create a tailored product export:
```
Comment (ghoussard) [inline content/md/imports-exports/tailored-import.md:30]: COPY PASTE SPOTTED!!!!! cc @ValentinMumble 👮🏻 

```suggestion
To create a Tailored product import:
```

File: content/md/imports-exports/tailored-import.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+1. Go to `Imports`
+1. Click on `Create import profile`
+1. Type a unique export code and a label, then select the job `Tailored product import`<!-- or `Tailored product model import`-->. <!--You can select either CSV or XLSX for each job.-->
+![Create a new product import profile](../img/Tailored-Import_Create-import-profile.png)
+1. Save your new import profile
+
+# Edit a tailored product import profile
+
+Once you've created your tailored import profile, or if you want to edit an existing one:
```
[the remark this answers is not in the document]
Comment (pierrejolly) [reply content/md/imports-exports/tailored-import.md:39]:
```suggestion
Once you've created your Tailored Import profile, or if you want to edit an existing one:
```

File: content/md/imports-exports/tailored-import.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+1. Click on the tailored import profile you want to edit
+1. Click on `Edit` located in the top right corner of the screen  
+
+# Discover the different tabs of this profile
+
+A tailored export profile is not far from a regular product and product import profile. The main difference is that you have a tab to configure the structure of your XLSX <!--or CSV -->file.
+
+Here are the tabs you can use:
+* `Properties`: To name your import profile.
+* `Permissions`: To set permissions to allow specific user groups to execute or edit your export profile (Enterprise Edition only).
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/md/imports-exports/tailored-import.md:50]: GE + EE :)

File: content/md/imports-exports/tailored-import.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+# Overview
+
+This feature consists in an import profile, available only for our SaaS users (Enterprise Edition or Growth Edition).
+It helps you to import your product information in XLSX or CSV files, with different structures than what the PIM usually accept.  
+For instance, let's assume your supplier or brands send you their product information in different XLSX files, having different structures:  
+* Supplier A sends a file with columns named: "Product ID", "Model Name", "Online description", and "Color".
+* Brand B sends a file with columns named: "Gencod, "Web title", "Long label", "Web description", "Supplied accessories".  
+
+**Without** Tailored Import, you have to import your product information thanks to a regular product import profile, and manually modify your spreadsheet to transform your attributes from code to label, transform you options values from label to codes, convert the measurements, and many more manual operations.
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/md/imports-exports/tailored-import.md:19]: Good point, thanks!

**Review by alexandrelissowski — APPROVED** [review]
few comments but it's very good.

**Pull request merged**